src/program.js

```diff
@@ -167,6 +167,11 @@ Example: $0 --help run.
         demand: true,
         type: 'string',
       },
+      'timeout' : {
+        describe: 'A length in millesconds to give up after if the add-on ' +
+                  'hasn\'t been signed',
```

Comment:
> Is this value undefined when not specified on the command line? It needs to be exactly undefined. It can't be false or null.

Verified: Works as expected. Logged the `timeout` using `typeof`. It is `undefined` when not specified.